src/cmd/tools/m3ctl/apply/apply.go

```diff
@@ -0,0 +1,19 @@
+package apply
+
+import (
+	"fmt"
+	"go.uber.org/zap"
+
+	"github.com/m3db/m3/src/cmd/tools/m3ctl/client"
+	"github.com/m3db/m3/src/cmd/tools/m3ctl/yaml"
+)
+
+func DoApply(endpoint string, filepath string, zapper *zap.Logger) error {
```

Comment:
All exported methods need docs or else fail lint, e.g. "// DoApply does X, Y, Z..."

Reply:
Run `make metalint` to check for these locally, this will fail CI unfortunately so needs fixing.